### DISCUSSION TITLE
Reuniones

### DISCUSSION THREAD

**Discussion by NadiaCruz1** [CodeSystem2022/Grupo_Programaci-nUTN-TercerSemestre#38] [Polls]
Debido a que las reuniones de Zoom suelen cortarse(no duran demasiado), creo que seria bueno implementar la utilización de Teams.
Este tiene una forma mas practica de trabajar, podemos realizar un grupo y subir archivos directamente a ese mismo grupo. Para que nuestra organizacion se mejor.

**Comment by NicolasBogado** [comment]
TEAMS se puede filmar los encuentros?

**Reply by NadiaCruz1** [reply]
Si, cuanta con esa función. 
Estuve verificando que tenga todas la herramientas que usaulamente usamos.

**Comment by ayelenlscabral** [comment]
Hola equipo!
Estuve viendo información sobre lo que dice nuestra scrum master de la semana y me parece bien que se implemente esta nueva metodología.

**Comment by EduMaxiMontes** [comment]
Bueno

**Comment by CamilaGonzalez97** [comment]
Me parece bien, para generar una mejor comunicación

**Comment by JoaquinZabala** [comment]
Estaría bueno también agregar Discord, ya que es una muy buena app y es fácil de usar!
